### PR TITLE
feat: Added support for clear undo redo stack.

### PR DESCRIPTION
**Pull request by LukasSefcik**
Added function clearing both undo a redo stack.
New test verifying undo redo functionality.
Updated README.md.

### CODE REVIEW

File: libs/ngrx-toolkit/src/lib/with-undo-redo.spec.ts [hunk 1]
```diff
@@ -190,5 +191,33 @@ describe('withUndoRedo', () => {
         expect(store.canRedo()).toBe(true);
       });
     }));
+
+    it('clears undo redo stack', fakeAsync(() => {
+      TestBed.runInInjectionContext(() => {
```
Comment (rainerhahnekamp) [inline libs/ngrx-toolkit/src/lib/with-undo-redo.spec.ts:196]: I know that you did it as the other tests before, but could please do it without runinInjectionContext? So you setup the signalStore first (with providedIn root), and then you just say TestBed.inject(Store)?

File: libs/ngrx-toolkit/src/lib/with-undo-redo.spec.ts [hunk 2]
```diff
@@ -190,5 +191,33 @@ describe('withUndoRedo', () => {
         expect(store.canRedo()).toBe(true);
       });
     }));
+
+    it('clears undo redo stack', fakeAsync(() => {
```
Comment (rainerhahnekamp) [inline libs/ngrx-toolkit/src/lib/with-undo-redo.spec.ts:195]: `fakeAsync` is going to be not available in zoneless and it is not required in this test. If you remove the `tick`, the test should still work.

Out of interest, why do you see the need for a `tick`?

**Review by rainerhahnekamp — CHANGES_REQUESTED** [review]
@LukasSefcik well done, two remarks and then we are ready to merge.

File: libs/ngrx-toolkit/src/lib/with-undo-redo.spec.ts (hunk 2 above)
Comment (LukasSefcik) [reply libs/ngrx-toolkit/src/lib/with-undo-redo.spec.ts:195]: You are absolutely right. No need to have `fakeAsync` and `tick`. I used `tick` automatically based on my previous experiences with some not working tests in my projects. Sorry for this.

File: libs/ngrx-toolkit/src/lib/with-undo-redo.spec.ts (hunk 1 above)
Comment (LukasSefcik) [reply libs/ngrx-toolkit/src/lib/with-undo-redo.spec.ts]: It's fixed. Thank you

**Review by rainerhahnekamp — APPROVED** [review]

**Comment by rainerhahnekamp** [comment]
Thanks @LukasSefcik

**Pull request merged**